esmvaltool/recipes/cmorizers/recipe_daily_era5.yml

```diff
@@ -0,0 +1,155 @@
+# ESMValTool
+# recipe_daily_era5.yml
+---
+documentation:
+  description: Create daily ERA5 data
+
+  authors:
+    - andela_bouwe
+    - kalverla_peter
+
+  references:
```

Comment:
any valid reference you may add here?

Reply:
ah my apologies, didn't see it's a draft - was also about to ask where's the refactored script :grin:

Reply:
no problem at all, actually it is almost ready. The documentation needs to be revised/added. I am wondering if there is already documentation for the file changed in this PR. I couldn't find any in the [Available recipes] section(https://docs.esmvaltool.org/en/latest/recipes/index.html#). 
Also, I edited the title of the pull request. :-)

Reply:
oh I found some docs for recipe_era5.yml [here](https://docs.esmvaltool.org/en/latest/input.html#using-a-cmorizer-script).

Reply:
> any valid reference you may add here?

Good question, @bouweandela and @Peter9192 do you know any reference for converting ERA5 hourly data to daily data? thanks.

Reply:
All the recipe does is some straightforward time aggregation. So if there's anything to cite, it would be the original dataset. Perhaps you could refer to the [ERA5 reference paper](https://rmets.onlinelibrary.wiley.com/doi/full/10.1002/qj.3803)?

Reply:
The reference is added.